Formula/n/navidrome.rb

```diff
@@ -0,0 +1,32 @@
+class Navidrome < Formula
+  desc "Modern Music Server and Streamer compatible with Subsonic/Airsonic"
+  homepage "https://www.navidrome.org"
+  url "https://github.com/navidrome/navidrome.git",
+      tag:      "v0.51.1",
+      revision: "6d253225de088e27f9a191d6bcedcb2e68b123fa"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+  depends_on "node@18" => :build
+  depends_on "pkg-config" => :build
+  depends_on "taglib"
```

Comment:
Don't you need this?
```suggestion
  depends_on "pkg-config" => :build
  depends_on "ffmpeg"
  depends_on "taglib"
```

Reply:
fixed